### PR TITLE
IfcFooting Rail Import BIMQ 2021-07-21T07:05:54.9322391Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcFooting

### CODE REVIEW

File: IFC4x3/Properties/i/IsReinforced_1Qp9XSGSD30AXUnzycyqDf/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+To indicate whether the OCS foundation is reinforced or not.
```
Comment (christophe-blt) [inline IFC4x3/Properties/i/IsReinforced_1Qp9XSGSD30AXUnzycyqDf/Documentation.md:1]: Propose documentation " True if the OCS foundation  is reinforced.

**Review by christophe-blt — COMMENTED** [review]
To edit

File: IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" PrimaryDataType="IfcLabel" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml:2]: Ik_code issue / Out of railway scope.

File: IFC4x3/Properties/i/IsReinforced_1Qp9XSGSD30AXUnzycyqDf/Documentation.md (hunk 1 above)
Comment (fhulin) [reply IFC4x3/Properties/i/IsReinforced_1Qp9XSGSD30AXUnzycyqDf/Documentation.md]: I have changed the documentation. Commit : a1b0cae8ebd1da839393bbf5ab16f2d1f2f833c2

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM, remove load bearing capacity changes as PR#185 has the correct accepted changes

**Review by SergejMuhic — COMMENTED** [review]
Are we keeping `IsReinforced`?

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**